syslinux.cfg.example

```diff
@@ -0,0 +1,11 @@
+DEFAULT webboot
+  SAY Now booting webboot
+PROMPT 1
+TIMEOUT 1
+
+LABEL webboot
+  KERNEL /boot/webboot
+  INITRD /boot/webboot.cpio.gz
```

Comment:
did you rename the kernel and initrd?

Reply:
This is on the boot media. setting up a boot directory is a common convention on the boot media.